telegram/ext/_contexttypes.py

```diff
@@ -53,6 +53,19 @@ class ContextTypes(Generic[CCT, UD, CD, BD]):
 
     """
 
+    DEFAULT_TYPE = CallbackContext["ExtBot", Dict, Dict, Dict]  # type: ignore[misc]
```

Comment:
actually, I think we can just use `dict` here instead of `typing.Dict` since we don't need to specify the types for keys & values